packages/dak-components/lib/crescat.ts

```diff
@@ -0,0 +1,97 @@
+import axios, { AxiosRequestHeaders } from 'axios'
+
+import cache from 'memory-cache';
+
+// Fetches the data from a given URL
+const cachedGet = async (url: string, headers: AxiosRequestHeaders) => {
+    const cachedResponse = cache.get(url);
+    if (cachedResponse) {
+        return cachedResponse;
+    }
+
+    const cacheTime = 1000 * 60 * 30; //Cached for 1 minutes
+    const response = await axios.get(url, {
+        headers
+    });
+
+    // Establishes the data from the URL-response
+    const data = response.data;
+    cache.put(url, data, cacheTime);
+    return data;
+}
+
+
+const getEvents = async () => {
+
+    // Fetches json payload from the Crescat booking site
+    const crescatData = await cachedGet("https://app.crescat.io/external/v1/calendar", {
+        Authorization: `Bearer ${process.env.CRESCAT_TOKEN}`
+    });
+
+    var rawShowTimes = Array();
+
+    // Cycles through all events in crescatData and retrieves showtimes
+    for (var ievent = 0; ievent < crescatData.length; ievent++) {
+
+        rawShowTimes.push(crescatData[ievent].show_times)
+    };
+
+    // Array to be returned
+    var jsonShowTimes = Array();
+
+    // Retrieves the json-elements from raw showtimes array
+    for (var stIndex = 0; stIndex < rawShowTimes.length; stIndex++) {
+
+        for (var stIndex2 = 0; stIndex2 < rawShowTimes[stIndex].length; stIndex2++) {
+            if (rawShowTimes[stIndex] != []) {
+                jsonShowTimes.push(rawShowTimes[stIndex][stIndex2]);
+            };
+        };
+    };
+
+    return jsonShowTimes;
+
+    //return crescatData.filter(x => x.fields.some(x => x.id === 70879));
+}
```

Comment:
What is the actual output here?

`if (rawShowTimes[stIndex] != [])` => Will always return true 
![image](https://user-images.githubusercontent.com/5204006/195795058-5014d83b-e806-44de-91a1-468fba0197cc.png)

Can actually be simplified down to:
```suggestion
const getEvents = async () => {
  // Fetches json payload from the Crescat booking site
  const crescatData = (await cachedGet(
    "https://app.crescat.io/external/v1/calendar",
    {
      Authorization: `Bearer ${process.env.CRESCAT_TOKEN}`,
    }
  )) as CrescatEvent[];

  const showTimes = crescatData.flatMap(event => event.show_times);
  return showTimes;
  
    //return crescatData.filter(x => x.fields.some(x => x.id === 70879));
}
```